src/targets/segment/segment.js

```diff
@@ -2,8 +2,9 @@
  * An function that sends an event to segment.io
  */
 function sendSegmentEvent(events) {
-  if (!window || !window.analytics) {
-    return;
+  if (!window) return;
+  if (window && !window.analytics) {
```

Comment:
`window` will alway be truethy when it gets to this line, because of the check on line 5. So this can simply be:
```js
if (!window.analytics) {
}
```